infra/ansible/inventory.yml

```diff
@@ -26,6 +26,10 @@ all:
           nvm_version: "v0.38.0"
           npm_lts_version: fermium
           frontend_scheme: http
+          frontend_csp_connect_src: https://noembed.com
+          frontend_csp_frame_src: https://www.youtube.com
+          frontend_csp_image_src: "https://i.ytimg.com https://www.paypal.com https://www.paypalobjects.com"
+          frontend_csp_script_src: "https://www.youtube.com/iframe_api https://www.youtube.com/s/player/"
```

Comment:
Perhaps it would be reasonable to simply set `https://www.youtube.com` here, for easier maintenance and to be more robust against potential changes in Youtube implementation?

Reply:
The YouTube domain is indeed more maintainable than a list of authorized URLs.

I was about to update the PR and I was discouraged to use directly `https://www.youtube.com` by a [CSP evaluator](https://csp-evaluator.withgoogle.com/) and this [particular slide](https://lbrunet.github.io/Plongee-dans-les-CSP/#32) of a  [CSP conference](https://lbrunet.github.io/Plongee-dans-les-CSP/#2). 

> /!\  www.youtube.com is known to host JSONP endpoints which allow to bypass this CSP.

See:

![capture](https://user-images.githubusercontent.com/39056254/146178783-255e1b29-5303-4d27-b8b7-1c243db93c2f.png)

I'm not familiar with JSONP, so I can't guarantee that `https://www.youtube.com/s/player/` doesn't also provide JSONP endpoints as `https://www.youtube.com` does.

I propose keeping the CSP as is for now, because:
- regarding the evaluator it might help mitigating the risk of JavaScript attacks
- if the YouTube ecosystem evolves, it will be easy for us to quickly update the CSP to update the authorized scripts
- if the YouTube ecosystem evolves too often, we can easily revert this decision of keeping the CSP as is

What do you think @amatissart?

Reply:
Ok. Thanks for looking into this!